getting-started/exploring-enterprise.md

```diff
@@ -0,0 +1,62 @@
+# Exploring TimescaleDB Enterprise
+
+TimescaleDB currently offers three product lines: OSS, Community, and Enterprise. 
+The Enterprise offering features automated data management that enables hands-free data retention policies 
+and data re-ordering on disk. For an overview of how TimescaleDB product lines differ, 
+please take a look at our [Pricing][pricing] page.
+
+## Requesting a License Key
+
+It is easy to get started with TimescaleDB Enterprise. In fact, if you are already 
+running TimescaleDB Community, all you have to do is obtain a license key to unlock 
+all Enterprise capabilities. No software upgrade or downgrade is required. We offer a free 30-day license 
+that you can request [online][license]. The license will be sent to you via email. 
+
+To upgrade or obtain a full Enterprise license key, please [contact sales][contact-sales].
+
+## Activating TimescaleDB Enterprise
+
+Before you can activate TimescaleDB Enterprise, you must have a Timescale-Licensed version of 
+TimescaleDB up and running. Note that our standard binary releases are by default Timescale-Licensed. 
+
+>:WARNING: If you explicitly built or downloaded an Apache 2.0 binary, you will not be able to 
+directly activate TimescaleDB Enterprise. You must upgrade to a Timescale-Licensed binary first. 
+Please note that this also applies to any TimescaleDB versions prior to v1.2.0.
+
+>:TIP: You can also set your license key prior to starting TimescaleDB by including 
```

Comment:
if you set license previously with alter system then setting it in postgresql.conf will not work

Reply:
@svenklemm should I just remove this then? Or add a tip that says that.

Reply:
@dianasaur323 or rather it will get overwritten by the one you set before with alter system, just wanted to make you  aware of it, dont feel strongly about adding another sentence to the tip, i'll let you decide

Reply:
whoops - I forgot about this and got excited when I saw the green button. Let me open another PR that does a slight fix to this